binder/postBuild

```diff
@@ -1 +0,0 @@
-python -m spacy download en
```

Comment:
we might still need them. @amn41 I guess binder is still running in our demo on the docs right?